src/interpreter/runtime/values/module.js

```diff
@@ -92,8 +92,26 @@ export function createInstance(
     },
 
     Memory({ node }: NodePath<Memory>) {
-      // TODO(sven): implement exporting a Memory instance
-      const memoryinstance = null;
+      const limits = {
```

Comment:
@xtuc it looks like limits are represented as strings within the AST - is that right? should they be `number`? also they should be able to handle hex representation

Reply:
If that's the case it's a bug, they should be number (or hex).

They are numbers in the AST
https://github.com/xtuc/js-webassembly-interpreter/blob/master/src/types/AST.js#L203-L204

Reply:
You are right! https://github.com/xtuc/js-webassembly-interpreter/blob/64851bba4ca86b1fa9e277099d0c3ffe13de6bfc/test/compiler/parsing/fixtures/watf/table/basic/expected.json#L21-L22

It's a bug.